common/opentelemetry-common.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
   spec.required_ruby_version = '>= 2.5.0'
 
-  spec.add_dependency 'opentelemetry-api', '~> 1.0.0.rc3'
+  spec.add_dependency 'opentelemetry-api', '~> 1.0.0'
```

Comment:
I wonder if this constraint should be loosened to `'~> 1.0'`? I'm no expert on semver, though.

Reply:
I believe that dropping the patch level (`~> 1.0`) will allow for optimistic dependencies allowing for any minor and patch `< 2.0`. 

E.g. this should allow someone to install `api@1.8.2` but continue using `common@1.0.0`.

Is that ok?

Reply:
In general, any backwards incompatible changes to the API or SDK packages after this **require** bumping the version to 2.0, so yes, that example should be OK.

Reply:
Will update to use `~> 1.0` for api and sdk dependencies.